tests/src/rules/newline-after-import.js

```diff
@@ -133,15 +133,30 @@ ruleTester.run('newline-after-import', require('rules/newline-after-import'), {
       options: [{ count: 2 }],
     },
     {
-      code: `import foo from 'foo';\n\n\nvar bar = 'bar';`,
+      code: `import foo from 'foo';\n\n// Some random comment\nvar bar = 'bar';`,
       parserOptions: { ecmaVersion: 2015, sourceType: 'module' },
       options: [{ count: 2, exactCount: true }],
     },
     {
-      code: `import foo from 'foo';\n\nvar bar = 'bar';`,
+      code: `import foo from 'foo';\n// Some random comment\nvar bar = 'bar';`,
```

Comment:
instead of changing existing test cases, let's add new ones :-)

Reply:
as you wish! new commit just pushed